client/resources/vorbis-stream.cpp

```diff
@@ -1,45 +1,45 @@
 // Copyright (c) 2013-2016 Trillek contributors. See AUTHORS.txt for details
 // Licensed under the terms of the LGPLv3. See licenses/lgpl-3.0.txt
 
-#include "resources/vorbis-stream.hpp"
+#include "vorbis-stream.hpp"
+
+#include <spdlog/spdlog.h>
 
-#include "spdlog/spdlog.h"
 #include "multiton.hpp"
 
 #undef STB_VORBIS_HEADER_ONLY
 
 // Rather than modifying the source to have these pragma just wrap the usages in the trillek code
 #if defined(_MSC_VER)
-	#pragma warning( push )
-	#pragma warning( disable : 4244) // Conversion loss of data
+#pragma warning( push )
+#pragma warning( disable : 4244) // Conversion loss of data
 
-	#pragma warning( push )
-	#pragma warning( disable : 4457) // declaration hides function parameter
+#pragma warning( push )
+#pragma warning( disable : 4457) // declaration hides function parameter
 
-	#pragma warning( push )
-	#pragma warning( disable : 4456) // declaration hides previous declaration
+#pragma warning( push )
+#pragma warning( disable : 4456) // declaration hides previous declaration
 
-	#pragma warning( push )
-	#pragma warning( disable : 4245) // signed / unsigned mismatch
+#pragma warning( push )
+#pragma warning( disable : 4245) // signed / unsigned mismatch
 
-	#pragma warning( push )
-	#pragma warning( disable : 4701) // potentially uninitialized variable
+#pragma warning( push )
+#pragma warning( disable : 4701) // potentially uninitialized variable
 
-	#include "resources/stb_vorbis.c"
+#include "stb_vorbis.c"
 
-	#pragma warning( pop ) 
-	#pragma warning( pop ) 
-	#pragma warning( pop ) 
-	#pragma warning( pop ) 
-	#pragma warning( pop )
+#pragma warning( pop ) 
+#pragma warning( pop ) 
+#pragma warning( pop ) 
+#pragma warning( pop ) 
+#pragma warning( pop )
 #else
-	#include "resources/stb_vorbis.c"
+#include "stb_vorbis.c"
```

Comment:
I think these should be indented to match the nested if/else structure.

Reply:
This is a good springboard to adding clang-format, yes?